lib/apollo-federation/tracing/proto/apollo_pb.rb

```diff
@@ -1,305 +1,55 @@
 # frozen_string_literal: true
 
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: apollo.proto
+# source: lib/apollo-federation/tracing/proto/apollo.proto
```

Comment:
This file was generated using:

```
protoc --ruby_out=./lib/apollo-federation/tracing/proto lib/apollo-federation/tracing/proto/apollo.proto
```

Since google protobuf no longer support ruby DSL: https://github.com/protocolbuffers/protobuf/commit/5cfc9e775d76acb37e5b554e53dff336a89d000f